components/SearchBar.tsx

```diff
@@ -25,23 +26,31 @@ const SearchBar = ({
   const [value, setValue] = React.useState(defaultValue || '');
   const intl = useIntl();
 
+  const handleClearFilter = () => {
+    setValue('');
+    onSubmit(null);
+  };
+
   // Reset value when `defaultValue` change, to handle reset filters
   React.useEffect(() => {
     setValue(defaultValue || '');
   }, [defaultValue]);
 
   return (
-    <SearchForm
-      placeholder={placeholder || intl.formatMessage(messages.searchPlaceholder)}
-      value={value}
-      onChange={setValue}
-      onSubmit={event => {
-        event.preventDefault();
-        const searchInput = event.target.elements.q;
-        onSubmit(searchInput.value || null);
-      }}
-      {...props}
-    />
+    <Box sx={{ maxWidth: '276px' }}>
```

Comment:
The component itself should not enforce any max width. This creates a visual regression on https://staging.opencollective.com/opencollective/updates.